poandy/controller/trade.py

```diff
@@ -0,0 +1,35 @@
+from poandy.util.request import RequestSender, RequestType
+from poandy.controller.base import Controller
+
+
+class TradeController(Controller):
+    @classmethod
+    def get_trades(cls, account_id, state="OPEN", instrument=None, count=50):
```

Comment:
I'm thinking, if we make it an enum, then we have to expose the enum type and the user has to pass an enum as argument, instead of string?

Reply:
User passing enum would be better imo because it kind of does away with invalid argument exception. if you pass a string, still need to check its validity. Have a public enum class that user can use to pass in

Reply:
then maybe we can have public enum classes for all oanda data types? Not just for trade, but other endpoints also. But maybe it can be a separate PR feature